common/sampling.h

```diff
@@ -131,12 +131,14 @@ llama_token llama_sampling_sample(
         struct llama_context * ctx_cfg,
         int idx = 0);
 
-// returns the probability that token of given id will be sampled
-llama_token_data_array llama_sampling_probability_distribution(
+// Prepares and adjusts the set of token candidates for sampling based on penalties, biases, and sampling parameters.
+llama_token_data_array llama_sampling_configure_token_candidates(
```

Comment:
Let's go for something shorter:

```suggestion
llama_token_data_array llama_sampling_prepare(
```

Reply:
Addressed in [bb38278](https://github.com/ggerganov/llama.cpp/pull/6240/commits/bb38278e6addca5a1affb00334ea6955242c6465) 👍